### DISCUSSION TITLE
Changelog?

### DISCUSSION THREAD

**Discussion by nollinvoyd** [aclap-dev/video-downloadhelper#197] [Help and Q&A]
### Your question

Did I already ask this? If I did, I can't find a thread here.

All these beta updates are great, but if there a settings option that's been altered or added, I'd like to know.

### Addon & CoApp version and details

v8.2.1.2a1, v2.0.17

**Comment by paulrouget** [comment]
https://github.com/aclap-dev/video-downloadhelper/discussions/categories/announcements
